src/routes/InventoryDetail.js

```diff
@@ -94,42 +104,38 @@ const BreadcrumbWrapper = ({ entity, inventoryId, entityLoaded }) => (
 const Inventory = () => {
   const chrome = useChrome();
   const { inventoryId } = useParams();
+  // eslint-disable-next-line no-unused-vars
```

Comment:
sus. :)

Reply:
ahh youre right, for some reason I had tried wrapping it in {}. And yeah it was because of that